runtime/pom.xml

```diff
@@ -26,7 +26,6 @@
     <dependency>
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-oidc-client-filter</artifactId>
-      <scope>provided</scope>
```

Comment:
@ricardozanini  for the record, this leaves the extension prepared to work
1) with the client filter by not having to add any additional dependencies on the users
2) work well in native mode

Reply:
I'd rather not bring an optional dependency if users won't use OAuth2. :(

Reply:
Honestly I don't see a huge problem with that, I fact, this is something happens many times with other let's say more high-level extensions that behind the scene they use other extensions.